test/test_bulk_update.py

```
@@ -383,6 +384,115 @@ def test_compile_time_error(self):
         assert res.exit_code != 0
         assert "'undefined_identifier' not defined" in str(res.exception)
 
+    def test_query_file_multi_csv_nodes_then_relations(self):
+        """Validate per-file query mappings and node-first processing in multi-file mode."""
+        graphname = "query_file_multi_csv"
+        users_file = "/tmp/users_update.tmp"
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/users_update.tmp"

(S108)